airflow/example_dags/example_task_group_decorator.py

```diff
@@ -33,19 +33,19 @@ def task_start():
 
 
 @task
-def task_1(value):
+def task_1(value: int):
```

Comment:
```suggestion
def task_1(value: int) -> str:
```

Probably worthwhile to add annotations for return values as well. (Same applies everywhere.)

Reply:
Good point

Reply:
Done